acceptance/openstack/imageservice/v2/images_test.go

```diff
@@ -0,0 +1,26 @@
+// +build acceptance imageservice images
+
+package v2
+
+import (
+	"testing"
+
+	"github.com/gophercloud/gophercloud/acceptance/clients"
+	"github.com/gophercloud/gophercloud/acceptance/tools"
+)
+
+func TestImagesCreateDestroyEmptyImage(t *testing.T) {
+	client, err := clients.NewImageServiceV2Client()
+	if err != nil {
+		t.Fatalf("Unable to create an image service client: %v", err)
```

Comment:
as an aside: `%v` here was always a mistake. When I did the initial cleanup of the tests, I had meant this to be `%#v`, but without understanding the structure of the error. I had thought there would be good bit of info in the error structure to print out. Turns out `%#v` would have been a bad choice. `%v` could easily just be `%s`. Maybe I'll fix that globally one day.